grails-app/services/au/org/emii/portal/WpsService.groovy

```diff
@@ -27,4 +37,36 @@ class WpsService extends AsyncDownloadService {
     def onResponseSuccess = { resp, xmlReader ->
         return new groovy.xml.StreamingMarkupBuilder().bindNode(xmlReader).toString()
     }
+
+    def notifyViaEmail(params) {
```

Comment:
If I merge this, I need to make sure that either:
 * This action is mocked (i.e., doesn't send email)
 * Protected - cannot be accessed except by authorized computers

Alternatively we can protect it with apache, but we have to remember to do so...